### DISCUSSION TITLE
Filament layout not loading properly on web server

### DISCUSSION THREAD

**Discussion by nishant91919** [filamentphp/filament#14260] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Hi
I have uploaded the filament solution on the web server but it is not showing properly. Please see the screenshot, Thanks in Advance...
![image](https://github.com/user-attachments/assets/8e548c32-3de4-43b6-b5af-cb11a91b201e)

**Comment by leandrocfe** [comment]
`php artisan filament:upgrade`
?

**Comment by nishant91919** [comment]
Thanks.....
